src/components/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Flex, Image, Link } from "@hackclub/design-system";
+
+const Base = Flex.extend`
+    flex-direction: row;
+    justify-content: space-between;
+`;
+
+const Header = () => (
+    <Base>
+        <Link href="https://hackclub.com" target="_blank">
+            <Image src="/flag.svg" width={128} ml={[3, 4, 5]} />
```

Comment:
in case you're not aware, `width` has a shorthand available, `w`. not necessarily preferable, just noting